rskj-core/src/main/java/co/rsk/metrics/profilers/Profiler.java

```diff
@@ -28,6 +28,7 @@ enum PROFILING_TYPE {
         DB_WRITE,
         FILLING_EXECUTED_BLOCK,
         LEVEL_DB_INIT,
+        ROCKS_DB_INIT,
```

Comment:
just not to make duplicates for `DB_INIT`, `DB_CLOSE` and `DB_DESTROY`, do you think we could have one set of these metrics?

Reply:
Could you elaborate a little bit more?

Reply:
my point is that do we need `LEVEL_DB_INIT` and `ROCKS_DB_INIT`? or we could simply have one - `DB_INIT`? Especially when we have `DB_CLOSE` and `DB_DESTROY` without any prefix

Reply:
Addressed